scan/src/components/modal/submitTx/SubmitTxModal.re

```diff
@@ -178,7 +157,7 @@ module SubmitTxStep = {
             };
           ();
         }}>
-        <Text value="Next" weight=Text.Bold size=Text.Md />
+        <Text value="Next" weight=Text.Medium size=Text.Lg />
       </button>
```

Comment:
Could we change to `<Button/>` component?

Reply:
@pzshine I've changed to `<Button/>` but have to fix cypress testing later.